.github/workflows/release_mgbench_client.yaml

```diff
@@ -0,0 +1,63 @@
+name: "Mgbench Bolt Client Publish Docker Image"
+
+on:
+  workflow_dispatch:
+    inputs:
+      version:
+        description: "Mgbench bolt client version to publish on Dockerhub."
+        required: true
+      force_release:
+        type: boolean
+        required: false
+        default: false
+
+jobs:
+  mgbench_docker_publish:
+    runs-on: ubuntu-latest
+    env:
+      DOCKER_ORGANIZATION_NAME: memgraph
+      DOCKER_REPOSITORY_NAME: mgbench-client
```

Comment:
@gitbuda In the end, between benchgraph-client and mgbench-client, I decided that we go with `mgbench-client`since users will probably not going to interact with the client explicitly, and it will be hidden from them. We are adding it to docker hub. 
If somebody goes that deep, it will probably know all the context about mgbench and benchgraph. From the code perspective, `mgbench-client` is much better.